cmd/compute-domain-daemon/main.go

```diff
@@ -291,7 +287,31 @@ func check(ctx context.Context, cancel context.CancelFunc, flags *Flags) error {
 	return nil
 }
 
-// cwriteNodesConfig creates a nodesConfig file with IPs for nodes in the same clique.
+// writeIMEXConfig renders the config template with the pod IP and writes it to the final config file.
+func writeIMEXConfig(podIP string) error {
+	configTemplateData := IMEXConfigTemplateData{
+		IMEXCmdBindInterfaceIP: podIP,
```

Comment:
Did you define a goal (in English :-)) in terms of networking reachability? We need that, and should document it in code. Then we can evaluate this specific choice against the goal.